extensions/azuremonitor/src/utils.ts

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+//TODO: This is the same file from mssql. Move this into a common place.
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as os from 'os';
+import * as findRemoveSync from 'find-remove';
+import { promises as fs } from 'fs';
+
+const configTracingLevel = 'tracingLevel';
+const configLogRetentionMinutes = 'logRetentionMinutes';
+const configLogFilesRemovalLimit = 'logFilesRemovalLimit';
+const extensionConfigSectionName = 'azuremonitor';
+
+export function removeOldLogFiles(logPath: string, prefix: string): JSON {
+	return findRemoveSync(logPath, { age: { seconds: getConfigLogRetentionSeconds() }, limit: getConfigLogFilesRemovalLimit() });
+}
+
+export function getConfiguration(config: string = extensionConfigSectionName): vscode.WorkspaceConfiguration {
+	return vscode.workspace.getConfiguration(extensionConfigSectionName);
+}
+
+export function getConfigLogFilesRemovalLimit(): number | undefined {
+	let config = getConfiguration();
+	if (config && config[configLogFilesRemovalLimit]) {
+		return Number((config[configLogFilesRemovalLimit]).toFixed(0));
+	}
+	else {
+		return undefined;
+	}
+}
+
+export function getConfigLogRetentionSeconds(): number | undefined {
+	let config = getConfiguration();
+	if (config && config[configLogRetentionMinutes]) {
+		return Number((config[configLogRetentionMinutes] * 60).toFixed(0));
+	}
+	else {
+		return undefined;
+	}
+}
+
+export function getConfigTracingLevel(): string | undefined {
+	let config = getConfiguration();
+	if (config) {
+		return config[configTracingLevel];
+	}
+	else {
+		return undefined;
+	}
+}
+
+export function getLogFileName(prefix: string, pid: number): string {
+	return `${prefix}_${pid}.log`;
+}
+
+export function getCommonLaunchArgsAndCleanupOldLogFiles(logPath: string, fileName: string, executablePath: string): string[] {
+	let launchArgs: string[] = [];
+	launchArgs.push('--log-file');
+	let logFile = path.join(logPath, fileName);
+	launchArgs.push(logFile);
+
+	console.log(`logFile for ${path.basename(executablePath)} is ${logFile}`);
```

Comment:
Either use the output channel or remote, we don't need to clutter up the console logs with this kind of stuff